styling-and-appearance/styling-apperance-editing-control-templates.md

```diff
@@ -71,35 +72,44 @@ As an example if you need to add a rounded red Border around ItemsPresenter in t
 	                </Setter.Value>
 	            </Setter>
 	        </Style>
-	        
 	    </ResourceDictionary>
 	</Application.Resources>
-	{{endregion}}
+{{endregion}}
 
+**Figure 2** shows the resulting control.
 
+#### __Figure 2: RadListBox control with red border__
 
-And here is the result:
-![styling-apperance-editing-control-templates 2](images/styling-apperance-editing-control-templates_2.png)
+![RadListBox control with red border](images/styling-apperance-editing-control-templates_2.png)
 
 ## Extracting Control Templates Using Visual Studio

```

Comment:
We can include a note that the same approach could be used to extract a style with Blend